src/views/forms/AccountManagementForms/TestnetDepositForm.tsx

```diff
@@ -43,6 +44,12 @@ export const TestnetDepositForm = ({ onDeposit, onError }: DepositFormProps) =>
     }
   }, [subAccount]);
 
+  useEffect(() => {
```

Comment:
does this mean we should have this for `DepositForm` and `WithdrawForm` as well?

Reply:
yup they already do -- this was missing